torch/_export/utils.py

```diff
@@ -122,7 +122,7 @@ def default_from_dumpable_context(dumpable_context: DumpableContext) -> Context:
         else default_from_dumpable_context
     )
 
-    _register_pytree_node(
+    register_pytree_node(
```

Comment:
We should probably leave this one alone? I'm not sure if this causes internal problems. We can swap it over later.